mailpoet/tests/_support/DefaultsExtension.php

```diff
@@ -83,7 +83,7 @@ private function setupWooCommerce() {
 
     // other
     update_option('woocommerce_bacs_settings', ['enabled' => 'yes'], 'yes');
-    update_option('woocommerce_cod_settings', ['enabled' => 'yes'], 'yes');
+    update_option('woocommerce_cod_settings', ['enabled' => 'yes', 'enable_for_virtual' => 'yes'], 'yes');
```

Comment:
For the future, I would suggest including in the commit message why passing `'enable_for_virtual' => 'yes'` is needed now and it was not before.

Reply:
Sure, sorry about that. It looks like the new checkout experience requires it.